llvm/lib/Target/M68k/GISel/M68kCallLowering.h

```diff
@@ -64,8 +64,8 @@ struct M68kIncomingValueHandler : public CallLowering::IncomingValueHandler {
                            ISD::ArgFlagsTy Flags) override;
 };
 
-struct FormalArgHandler : public M68kIncomingValueHandler {
-  FormalArgHandler(MachineIRBuilder &MIRBuilder, MachineRegisterInfo &MRI)
+struct M68kFormalArgHandler : public M68kIncomingValueHandler {
```

Comment:
In addition to renaming to M68kFormalArgHandler, could we also move this to M68kCallLowering.cpp? I don't think it is exposed to other part of the backend.

Reply:
Ok, moved it to the cpp file.